distribution/docs/src/main/resources/content/_configuring/banner-markings-content-extractor.adoc

```diff
@@ -0,0 +1,89 @@
+:title: Banner Markings Extractor
+:type: configuringConfigFile
+:status: published
+:summary: Banner Markings Extractor
+:order: 132
+
+=== Banner Markings Extractor
+
+The Banner Markings Extractor parses the text of PDFs, Microsoft Office documents, plain text files,
+or any other file from which the <<_tika_input_transformer,TikaTransformer>> can extract text.
+The ${branding} default behavior is to apply high-water security settings when ingesting a document.
+This is sufficient for documents with no security markings, but is incorrect when working with
+classified documents that include security mark-up in the text.
+
+The Banner Markings Extractor parses a banner marking and attempts to create new attributes on the
+metacard of the ingested document. The extractor looks at the first non-blank line of text
+in the document and tries to interpret it as a banner marking. The interpretation of a banner
+marking is guided by DoD Instruction 5200.01 and the ${branding} catalog taxonomy. The marking extractors
+will attempt to get as much information from the banner marking as possible.
+The attributes they attempt to extract are:
+
+* Classification
+* Releasibility
+* Codewords
+* Dissemination Controls
+* Owner-Producer
+* Classification System
+* Special Access Program
+* Atomic Energy Act
+* DoD Unclassified Controlled Nuclear Information (DoD UCNI)
+* Department of Energy (DOE) UCNI
+* Foreign Government Information
+* Other dissemination markings
+
+[NOTE]
+====
+The Banner Markings Extractor looks at the first non-blank line of text in a document.
```

Comment:
❓ should we explain how this relates to 5200.01, and is not just an arbitrary limitation on our part?

Reply:
I agree. It's according to spec and we shouldn't expect all our users to have read the spec. We don't have to direct them to the precise line, but referencing it as our source of truth would help our users have more confidence in this code.

Reply:
Which part is related to 5200.1? The "first-line" part? (I haven't read much of 5200.01, just scanned it)

Reply:
That's the line.